test/clojure/core_test/constantly.cljc

```diff
@@ -0,0 +1,44 @@
+(ns clojure.core-test.constantly
+  (:require clojure.core
+            [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer)  [when-var-exists]]))
+
+(when-var-exists clojure.core/constantly
+  (deftest test-constantly
+    (testing "`constantly`"
+      (is (fn? (constantly nil)))
+
+      (are [v] (= v ((constantly v)))
+        `sym
+        "abc"
+        100000
+        1.0
+        #?(:clj 2/3) ; ratio not supported in cljs
```

Comment:
Should this be `:default`? I assume most Clojure variants support ratios and this is basically just a cljs issue

Reply:
Yep, I think CLJS is unique in not supporting ratios.

Reply:
I think we'd need an explicit `:cljs` for `:default` to work properly but we could just put something else here for `:cljs`

Reply:
I would do a `#js` object here for `:cljs` instead of the ratio

Reply:
could get cheeky and do
```clj
:cljs #js {"cljs does not have" "ratios"}
```